_includes/devdoc/bitcoin-core/rpcs/rpcs/sendmany.md

```diff
@@ -55,6 +55,16 @@ The `sendmany` RPC {{summary_sendMany}}
 
 {% enditemplate %}
 
+*Parameter #5---automatic fee subtraction
+
+{% itemplate ntpd1 %}
+- n: "Subtract Fee From Amount"
+  t: "boolean"
```

Comment:
In the `sendmany` call, this is an array of strings (addresses) rather than a single bool for the whole transaction.